src/compressor/Compressor.cc

```diff
@@ -27,6 +27,9 @@ const char * Compressor::get_comp_alg_name(int a) {
   case COMP_ALG_SNAPPY: return "snappy";
   case COMP_ALG_ZLIB: return "zlib";
   case COMP_ALG_ZSTD: return "zstd";
+#if defined(HAVE_QATZIP)
+  case COMP_ALG_QATZIP: return "qatzip";
+#endif
```

Comment:
(If it is compatible, then we should not have a different type. As it is the on-disk format basically requires intel quickassist to read data written this way...)